### PR TITLE
data: shrix1

### PR DESCRIPTION
**Pull request by shrix1**
Create Linkfree for the First time

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4758"></a>

### CODE REVIEW

File: data/shrix1.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Shriprasanna",
+  "type": "personal",
+  "bio": "Front-End developer based in India",
+  "avator": "https://github.com/shrix1.png",
```
Comment (priyankarpal) [inline data/shrix1.json:5]: Hey, @shrix1 currently you don't need to add this `avatar` code. Please remove this line.
``` suggestion
 
```

File: data/shrix1.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/shriprasanna007",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/shrix1",
+      "icon": "FaLinkedin"
+    },
```
Comment (priyankarpal) [inline data/shrix1.json:16]: Also, please delete this comma.
``` suggestion
    }
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/shrix1.json (hunk 2 above)
Comment (shrix1) [reply data/shrix1.json:16]: ok i'll do it

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Review by amandamartin-dev — APPROVED** [review]

**Pull request merged**